### DISCUSSION TITLE
Request for HLK-LD2451 Component for ESPHOME

### DISCUSSION THREAD

**Discussion by ayubn** [TillFleisch/ESPHome-HLK-LD2450#33] [Ideas]
hi can u please also create a code of new sensor hlk-ld2451

**Comment by TillFleisch** [comment]
Maybe some time in the future, if I have some sensors and find a use for them. No promises.

**Reply by ayubn** [reply]
thanks i am trying to covert its protocol file into  yaml code it start working but not as it should be
